module/preference/PreferenceDialog.kt

```diff
@@ -378,26 +407,23 @@ object PreferenceDialog {
         }
 
         class ProjectManager : PreferenceGroup(PROJECT_MANAGER) {
-            companion object {
-                private const val IGNORED_PATHS_PLACEHOLDER = ".git"
-            }
-
-            private val ignoredPathsString = preferenceMgr.ignoredPaths.joinToString(", ")
-            private var ignoredPaths = PreferenceField.TextInput(
+            private val ignoredPathsString = preferenceMgr.ignoredPaths.joinToString("\n")
+            private var ignoredPaths = PreferenceField.MultilineTextInput(
                 initValue = ignoredPathsString,
-                label = PROJECT_IGNORED_PATHS, placeholder = IGNORED_PATHS_PLACEHOLDER,
-                caption = IGNORED_PATHS_CAPTION
+                label = PROJECT_IGNORED_PATHS,
+                caption = IGNORED_PATHS_CAPTION,
+                fieldHeight = 5,
```

Comment:
Let's delete (and hardcode the height) for now - it's not intuitive what `fieldHeight` means (it certainly is not a 5-line multiline input)

Reply:
Done.